<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinGameWindow</class>
 <widget class="QDialog" name="JoinGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Join Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #2b2b2b;
}

QLabel {
    color: #ffffff;
}

QListWidget {
    background-color: #1e1e1e;
    color: #ffffff;
    border: 2px solid #404040;
    border-radius: 5px;
    padding: 5px;
}

QListWidget::item {
    padding: 10px;
    border-radius: 3px;
}

QListWidget::item:selected {
    background-color: #0d7377;
}

QListWidget::item:hover {
    background-color: #323232;
}

QPushButton {
    background-color: #0d7377;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 10px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #14a5a5;
}

QPushButton:pressed {
    background-color: #0a5a5e;
}

QPushButton:disabled {
    background-color: #404040;
    color: #808080;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>üèÅ Server List üèÅ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="instructionLabel">
     <property name="text">
      <string>Select a game from the list below and click "Join Game" to play</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="gamesListWidget">
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>üîÑ Refresh</string>
       </property>
       <property name="toolTip">
        <string>Refresh the list of available games</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="joinButton">
       <property name="text">
        <string>‚ñ∂ Join Game</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #444444;
}

QPushButton:hover {
    background-color: #555555;
}

QPushButton:pressed {
    background-color: #333333;
}</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
